updater_test.go

```diff
@@ -269,13 +269,11 @@ func TestUpdaterDownloadError(t *testing.T) {
 
 	require.NotNil(t, ctx.errReported)
 	assert.Equal(t, ctx.errReported.(Error).errorType, DownloadError)
-	assert.Equal(t, "deadbeef", ctx.updateReported.InstallID)
-	assert.Equal(t, "cafedead", ctx.updateReported.RequestID)
 	assert.False(t, ctx.successReported)
 }
 
 func TestUpdaterCancel(t *testing.T) {
-	testServer := testServerForError(t, fmt.Errorf("cancel"))
```

Comment:
If we want to test the prompt action flow, we now need to go through with getting a download first.